modules/util/kotlin/com/github/timaxinc/kcipt/util/io/String.kt

```diff
@@ -0,0 +1,27 @@
+package com.github.timaxinc.kcipt.util.io
```

Comment:
will only be used in BlockingClassLoader so it should be paced inside of  BlockingClassLoader class